static/app/components/replays/videoReplayer.tsx

```diff
@@ -445,6 +445,12 @@ export class VideoReplayer {
   }
 
   protected setVideoTime(video: HTMLVideoElement, timeMs: number) {
+    // If 'ended' is true, the current time will be overwritten to 0 after hitting play.
+    // This ensures that ended is reset to false so current time doesn't get overwritten
```

Comment:
Not sure I follow here, how do we ensure that ended is reset to `false`?

Reply:
Ah I re-read the PR and the description there makes sense. Let's include some of that information here:

1) Setting `currentTime` will cause a side-effect of resetting `ended` to false.
2) The additional assignment of `currentTime` is required to make sure `ended` is reset before we assign the actual `currentTime`
3) we set it to 1ms before to reduce flickering